assets/sass/components/dashboard/_googlesitekit-DashboardPageSpeed.scss

```diff
@@ -37,21 +37,41 @@
 		}
 	}
 
+	.googlesitekit-pagespeed-widget__content-wrapper {
+		position: relative;
+	}
+
+	.googlesitekit-pagespeed-widget__content-wrapper--loading {
+		.googlesitekit-pagespeed-widget__content {
+			visibility: hidden;
+		}
+	}
+
+	.googlesitekit-pagespeed-widget__progress-bar {
+		left: 50%;
+		position: absolute;
+		top: 50%;
+		transform: translate(-50%, -50%);
+		width: 100%;
+	}
+
 	.googlesitekit-pagespeed-widget__header {
-		align-items: flex-end;
 		border-bottom: 1px solid $c-utility-divider;
 		border-radius: $br-sm $br-sm 0 0;
 		display: flex;
-		justify-content: space-between;
+		flex-wrap: wrap-reverse;
+		justify-content: center;
 
-		@media (max-width: $bp-xsmallOnly) {
-			align-items: center;
-			flex-direction: column-reverse;
-			justify-content: flex-end;
+		.mdc-tab-scroller__scroll-content {
+			@media (min-width: $bp-tablet) {
+				flex: 0;
+			}
 		}
 	}
 
 	.googlesitekit-pagespeed-widget__data-src-tabs {
+		align-items: flex-end;
+		flex-grow: 1;
```

Comment:
These tweaks are needed to accommodate width of the new "How to improve" tab in the widget header.